test/test_integration_pumactl.rb

```diff
@@ -64,7 +64,7 @@ def test_phased_restart_cluster
 
     start = Process.clock_gettime(Process::CLOCK_MONOTONIC)
 
-    s = UNIXSocket.new @bind_path
+    s = TestPuma::SktUNIX.new @bind_path
```

Comment:
This kind of change is what I'm not thrilled about. Are you saying that these tests should be converted to use methods from `test/helpers/integration.rb` anyway?